azavea/pfb-network-connectivity
# Reformat Neighborhood display name in UI

## Overview

Codifies the pre-existing implicit assumption that state_abbrev be a part of Neighborhood.label for uniqueness constraints. Otherwise, how do we determine the whether we're talking about OR or ME for a neighborhood labeled 'Portland'? It's now no longer necessary to include the state as part of the label, since it can be added to a display string in a view layer via the state_abbrev field anyways.

## Testing Instructions

- Run migrations
- Ensure none of the labels in your dev box have state information in the label
- View UI

Once this is merged, a production deploy with migrations will be necessary. Once that is done, we'll have to manually update the neighborhood objects there to remove state from the labels. No updates to the batch shapefiles is necessary.

Connects #449
